examples/aio-sx/geo-location/geo-location.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: starlingx.windriver.com/v1
+kind: System
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: vbox
+  namespace: geo-location
+spec:
+  latitude: 45.35189954974955
+  longitude: -75.91866628453701
```

Comment:
I'm not sure the lack of a newline at the end of a file will impact the generation of the yaml file but it might be safer to add it in.